client/src/pages/Home.js

```diff
@@ -14,31 +12,25 @@ const Home = () => {
           Instant custom training plans that seamlessly sync to your calendar
             </div>
       </div>
-
-      <div style={{ display: "flex" }}>
-        <div style={{ width: "50%" }}>
-          <SideImage image={"/media/run1.jpg"} />
-        </div>
-        <div style={{ width: "50%" }}>
-          <SideText subtitle={"Custom"} text={"Whether you're going from couch to 5k or looking to set a Personal Record at your 10th marathon, every runner is unique and has different training needs. Jog Log's algorithm creates a custom training plan to suit your fitness level and schedule."} />
+      <div className="rowName">
```

Comment:
Change these into components, look at Hero for an example, instead of SideImage and SideText these can be a single component to make it more simple.